Formula/n/nanopb-generator.rb

```diff
@@ -17,7 +17,8 @@ class NanopbGenerator < Formula
   end
 
   depends_on "protobuf"
-  depends_on "python@3.11"
+  depends_on "python-setuptools"
```

Comment:
Is this needed for runtime, or can we make this a build-only dependenct?

Reply:
it is needed for runtime, see this run log, https://github.com/Homebrew/homebrew-core/actions/runs/6839004833/job/18596765896